BookEcommerce.Models/DAL/Repositories/ProductVariantRepository.cs

```diff
@@ -15,5 +15,16 @@ public class ProductVariantRepository : Repository<ProductVariant> , IProductVar
         public ProductVariantRepository(DbFactory dbFactory) : base(dbFactory)
         {
         }
+
+        public async Task<ProductVariant> GetProductVariantById(Guid idProductVariant)
+        {
+            return await GetQuery(pv => pv.ProductVariantId.Equals(idProductVariant)).FirstAsync();
+        }
```

Comment:
I don't think we have 2 records with same Id. So using single/singleOrDefault instead
btw, please update all param idProductVariant => productVariantId, idProduct....

Reply:
yes, I will fix it